examples/kieservers/client-cert-vault.yml

```diff
@@ -0,0 +1,9 @@
+kieservers:
+  - host: http://localhost:18080/kie-server/services/rest/server
+client-cert:
+    cert-name: pim
+    cert-credentials-provider: quarkus.file.vault.provider.pim.cert
+    keystore-path: /path/to/keystore.jks
+    keystore-credentials-provider: quarkus.file.vault.provider.pim.keystore
+    truststore-path: /path/to/truststore.jks
+    truststore-credentials-provider: quarkus.file.vault.provider.pim.truststore
```

Comment:
It would be nice and clearer if you could also add the properties in yaml for the `quarkus.file.vault.provider` entries